sharktank/sharktank/types/tensors.py

```diff
@@ -869,6 +869,14 @@ def __init__(
         super().__init__(name=name, shape=shape, layout_type=type(layout))
         self.layout = layout
 
+    def to(self, *args, **kwargs) -> "PlanarQuantizedTensor":
```

Comment:
This doesn't belong in this PR, should be seperate.

Reply:
Moved it to a new PR https://github.com/nod-ai/shark-ai/pull/2376.